docs/static/releasenotes.asciidoc

```diff
@@ -32,6 +32,10 @@ coming[8.3.0]
 [[notable-8.3.0]]
 ==== Performance improvements and notable issues fixed
 
+* {ls} is more efficient at fetching pipelines. 
+When a {ls} instance sends its pipeline IDs to {es} or central pipeline management in {kib}, it gets back only the pipeline configs that belong to that instance. 
++ 
+These improvements dramatically decrease network load while also giving users the ability to control pipelines dynamically using wildcards.
```

Comment:
ToDo: Add links to appropriate PR number(s).

Reply:
Here are the PRs:
- [ES PR](https://github.com/elastic/elasticsearch/pull/85847)
- [LS PR](https://github.com/elastic/logstash/pull/14076)